astarte-transport/astartetransport.cpp

```diff
@@ -654,4 +654,14 @@ void AstarteTransport::routeWave(const Hyperspace::Wave &wave, int fd)
     Q_EMIT waveReceived(interface, w);
 }
 
+bool AstarteTransport::connectToBroker()
+{
+  return m_mqttBroker->connectToBroker();
```

Comment:
`m_mqttBroker` could be `null`, check with `isNull()` before calling methods on it